aw/ViewControllers/Reaches/Filters/FilterDistanceViewController.swift

```diff
@@ -18,6 +18,16 @@ class FilterDistanceViewController: UIViewController {
     var distance: Float! {
         didSet {
             setDistanceLabel()
+
+            if distance > 0, let parent = self.parent {
```

Comment:
You may want to add this as a method on the parent, rather than accessing the child this way. Something like `parent.clearRegions`. I'll leave this up to you though.